jazz_codeq/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "api-nodejs-template",
+  "version": "0.0.1",
+  "description": "Nodejs Template project",
+  "main": "index.js",
+  "private": true,
+  "author": [],
+  "scripts": {
+    "test": "mocha"
+  },
+  "dependencies": {
+    "request": "*",
+    "deasync": "*",

```

Comment:
do we need deasync ?

Reply:
will take a look if it was being used somewhere